src/main/kotlin/no/nav/tiltakspenger/datadeling/motta/infra/db/DataSourceSetup.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.tiltakspenger.datadeling.motta.infra.db
+
+import com.zaxxer.hikari.HikariDataSource
+
+object DataSourceSetup {
+
+    fun createDatasource(url: String): HikariDataSource {
+        return HikariDataSource().apply {
+            jdbcUrl = url
```

Comment:
Ser du har oppsett for lokal kjøring i `FlywayMigrate.kt` og skal det kjøre lokalt må man vel ha `username` og `password` her også? Samme som vi gjorde i TP-saksbehandling-api på onsdag